index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/css/normalize.css">
+    <link rel="stylesheet" href="/css/styles.css">
+    <title>PC Store</title>
+</head>
+
+<body>
+    <div class="bar">
+        <header>
+            <div class="logo">
+                <img class="logo-image" src="/img/logo.png" alt="">
+                <p class="logo-name">GAMING WORLD</p>
```

Comment:
@maury18 it would be better to explain more why you think that h1 is right one as the title as the previous reviewer asked you ,so please try to change your comment with more details